[main] `fsharp` building failed with 3 new packages used not in baseline

[Failing build](https://dev.azure.com/dnceng/internal/_build/results?buildId=2555134) (internal Microsoft link)

`Alpine320_Online_MsftSdk_x64` leg:
[baseline-comparison.xml](https://artprodcus3.artifacts.visualstudio.com/Ab55de4ed-4b5a-4215-a8e4-0a0a5f71e7d8/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2RuY2VuZy9wcm9qZWN0SWQvN2VhOTExNmUtOWZhYy00MDNkLWIyNTgtYjMxZmNmMWJiMjkzL2J1aWxkSWQvMjU1NTEzNC9hcnRpZmFjdE5hbWUvQWxwaW5lMzIwX09ubGluZV9Nc2Z0U2RrX3g2NF9CdWlsZExvZ3NfQXR0ZW1wdDE1/content?format=file&subPath=%2Fartifacts%2Fprebuilt-report%2Fbaseline-comparison.xml)
[generated-new-baseline.xml](https://artprodcus3.artifacts.visualstudio.com/Ab55de4ed-4b5a-4215-a8e4-0a0a5f71e7d8/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2RuY2VuZy9wcm9qZWN0SWQvN2VhOTExNmUtOWZhYy00MDNkLWIyNTgtYjMxZmNmMWJiMjkzL2J1aWxkSWQvMjU1NTEzNC9hcnRpZmFjdE5hbWUvQWxwaW5lMzIwX09ubGluZV9Nc2Z0U2RrX3g2NF9CdWlsZExvZ3NfQXR0ZW1wdDE1/content?format=file&subPath=%2Fartifacts%2Fprebuilt-report%2Fgenerated-new-baseline.xml)

    /vmr/repo-projects/Directory.Build.targets(778,5): warning : 3 new packages used not in baseline! See report at /vmr/artifacts/prebuilt-report/baseline-comparison.xml for more information. Package IDs are: [/vmr/repo-projects/dotnet.proj]
    /vmr/repo-projects/Directory.Build.targets(778,5): warning : Microsoft.AspNetCore.App.Runtime.linux-musl-x64.9.0.0-rc.1.24452.1 [/vmr/repo-projects/dotnet.proj]
    /vmr/repo-projects/Directory.Build.targets(778,5): warning : Microsoft.NETCore.App.Crossgen2.linux-musl-x64.9.0.0-rc.1.24431.7 [/vmr/repo-projects/dotnet.proj]
    /vmr/repo-projects/Directory.Build.targets(778,5): warning : Microsoft.NETCore.App.Runtime.linux-musl-x64.9.0.0-rc.1.24431.7 [/vmr/repo-projects/dotnet.proj]
      3 packages in baseline weren't used!
      10 packages used as expected in the baseline.

`Ubuntu2404Arm64_Offline_MsftSdk_arm64` leg:

      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: Unable to find package Microsoft.NETCore.App.Runtime.linux-arm64 with version (= 9.0.0-rc.1.24431.7) [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Found 1 version(s) in previously-source-built [ Nearest version: 9.0.0-preview.7.24405.7 ] [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from /vmr/.dotnet/sdk/9.0.100-rc.1.24452.12/FSharp/library-packs were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from prebuilt were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from reference-packages were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from source-built-transport-arcade were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: Unable to find package Microsoft.NETCore.App.Crossgen2.linux-arm64 with version (= 9.0.0-rc.1.24431.7) [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Found 1 version(s) in previously-source-built [ Nearest version: 9.0.0-preview.7.24405.7 ] [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from /vmr/.dotnet/sdk/9.0.100-rc.1.24452.12/FSharp/library-packs were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from prebuilt were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from reference-packages were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from source-built-transport-arcade were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: Unable to find package Microsoft.AspNetCore.App.Runtime.linux-arm64 with version (= 9.0.0-rc.1.24452.1) [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Found 1 version(s) in previously-source-built [ Nearest version: 9.0.0-preview.7.24406.2 ] [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from /vmr/.dotnet/sdk/9.0.100-rc.1.24452.12/FSharp/library-packs were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from prebuilt were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from reference-packages were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]
      /vmr/src/fsharp/buildtools/fslex/fslex.fsproj : error NU1102: - Versions from source-built-transport-arcade were not considered [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj]

--- a/src/SourceBuild/content/eng/Versions.props
+++ b/src/SourceBuild/content/eng/Versions.props
@@ -25,7 +25,7 @@
     -->
     <PrivateSourceBuiltSdkVersion>9.0.100-preview.7.24407.1</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>9.0.100-preview.7.24407.1</PrivateSourceBuiltArtifactsVersion>
-    <PrivateSourceBuiltPrebuiltsVersion>10.0.100-alpha.1.24508.1</PrivateSourceBuiltPrebuiltsVersion>
+    <PrivateSourceBuiltPrebuiltsVersion>0.1.0-10.0.100-1</PrivateSourceBuiltPrebuiltsVersion>
     <!-- command-line-api dependencies -->
     <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
     <!-- msbuild dependencies -->

--- a/src/SourceBuild/content/eng/Versions.props
+++ b/src/SourceBuild/content/eng/Versions.props
@@ -25,7 +25,7 @@
     -->
     <PrivateSourceBuiltSdkVersion>9.0.100-preview.7.24407.1</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>9.0.100-preview.7.24407.1</PrivateSourceBuiltArtifactsVersion>
-    <PrivateSourceBuiltPrebuiltsVersion>10.0.100-alpha.1.24503.1</PrivateSourceBuiltPrebuiltsVersion>
+    <PrivateSourceBuiltPrebuiltsVersion>10.0.100-alpha.1.24508.1</PrivateSourceBuiltPrebuiltsVersion>
     <!-- command-line-api dependencies -->
     <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
     <!-- msbuild dependencies -->

